.github/workflows/integration_test.yaml

```diff
@@ -0,0 +1,44 @@
+name: Integration tests
```

Comment:
What was the reason for not using the `operator-workflow` here please?

Reply:
I believe it's because we need to set Devstack Swift like Wordpress charm.

Reply:
`setup-devstack-swift` is an argument to the integration tests, would that work? https://github.com/canonical/operator-workflows/blob/main/.github/workflows/integration_test.yaml

Reply:
Now it does! :-)